src/DisplayBoard.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <sstream>
+#include <iostream>
+#include "Board.hpp"
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b);
+}
+
+//alias to shortcut name in test files
+namespace DB = DisplayBoard;
```

Comment:
There is no advantage in using this alias, instead in test you need to check in other file what DB stands for.

Reply:
Ok, I wanted to use it in test class by I can removed it.